index.js

```diff
@@ -54,7 +73,17 @@ function getHighestMetascore() {}
  *  getAverageIMDBRating(movies);
  *  //> 7.76
  */
-function getAverageIMDBRating() {}
+function getAverageIMDBRating(movies) {
+   let average = 0;
+   if (movies.length === 0) {
+     return average;
+   }
+   for (let i = 0; i < movies.length; i++) {
+     average = average + Number(movies[i].imdbRating);
+   }
+
+  return Number((average / movies.length).toFixed(2)) ;
+}
```

Comment:
I love the toFixed() call just in case we have extra decimals.  Nice work.  I would lose the empty line on 84 though as this code is very easy to follow.